src/main_gui.cpp

```diff
@@ -248,17 +248,17 @@ struct MainWindow : Window
 		this->DrawWidgets();
 		if (_game_mode == GM_MENU) {
 			static const SpriteID title_sprites[] = {SPR_OTTD_O, SPR_OTTD_P, SPR_OTTD_E, SPR_OTTD_N, SPR_OTTD_T, SPR_OTTD_T, SPR_OTTD_D};
-			static const uint LETTER_SPACING = 10;
-			int name_width = (lengthof(title_sprites) - 1) * LETTER_SPACING;
+			uint letter_spacing = UnScaleByZoom(40, _gui_zoom);
```

Comment:
There is `ScaleGUITrad` to scale traditional pixel values. It will also work in the future, when the base-zoom-level is changed again for 8k screens.
```suggestion
			uint letter_spacing = ScaleGUITrad(10);
```

Reply:
This is great, thank you for the pointer.